File: BE/API/Models/Presupuesto.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace API.Models;

public class Presupuesto
{ 
    public Guid Id { get; set; }
    public string? campania { get; set; }
    public double Total { get; set; }
    // public Guid? ClienteId { get; set; }
    public bool Activo { get; set; }
    public bool Aceptado { get; set; }
    public Cliente? Cliente { get; set; }
    public DateTime Fecha { get; set; }

    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int Numero { get; set; }
}

File: FE/src/interfaces/presupuesto.ts
import { Guid } from "guid-typescript";
import { Cliente } from "./cliente";
import { DetallePresupuesto } from "./detallePresupuesto";
import { Producto } from "./producto";

export interface Presupuesto {
    Id: string,
    Numero: string,
    Cliente: Cliente,
    Campania: string,
    total: number,
    activo: boolean,
    aceptado: boolean,
    productos?: Producto[]
}